sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubListener.PartitionProcessor.cs

```diff
@@ -188,7 +185,7 @@ public async Task ProcessEventsAsync(EventProcessorHostPartition context, IEnume
                             if (_cachedEventsBackgroundTaskCts == null && CachedEventsManager.HasCachedEvents)
                             {
                                 // If there are events waiting to be processed, and no background task running, start a monitoring cycle.
-                                _cachedEventsBackgroundTaskCts = CancellationTokenSource.CreateLinkedTokenSource(_cts.Token);
+                                _cachedEventsBackgroundTaskCts = CancellationTokenSource.CreateLinkedTokenSource(linkedCts.Token);
```

Comment:
If linkedCts is disposed, won't this go boom?  Do we want to create a link using the three source tokens?

Reply:
How would it be disposed? The underlying token would only get disposed after StopProcessing finishes I thought.

Reply:
Ah I see your point..